web-api/src/models.rs

```diff
@@ -219,19 +223,17 @@ impl DocumentProperty {
                         });
                     };
                     trim(string);
-                    if !is_valid_string(string, 2_048) {
+                    let Ok(()) = check_string_property_value(string) else {
                         return Err(InvalidDocumentProperty {
-                            document: document_id.clone(),
                             property: property_id.clone(),
                             invalid_value: value.clone(),
                             invalid_reason: InvalidDocumentPropertyReason::InvalidString,
                         });
-                    }
+                    };
```

Comment:
:art: semicolon can be removed if above's suggestion is used
```suggestion
                }
```